src/service.js

```diff
@@ -0,0 +1,316 @@
+if(!global._babelPolyfill) { require('babel-polyfill'); }
+
+import Proto from 'uberproto';
+import mongoose from 'mongoose';
+import filter from 'feathers-query-filters';
+import errors from 'feathers-errors';
+import errorHandler from './error-handler';
+
+// Use native promises
+mongoose.Promise = global.Promise;
```

Comment:
I think the times for being that flexible about a Promise library are over. We're using ES6 so you'll get native promises. The Polyfill will always provide them and Node 5 comes with them already anyway (I think).